stdlib/InteractiveUtils/src/InteractiveUtils.jl

```diff
@@ -308,36 +307,23 @@ function dumpsubtypes(io::IO, x::DataType, m::Module, n::Int, indent)
     nothing
 end
 
-const Distributed_modref = Ref{Module}()
-
+# TODO: @deprecate peakflops to LinearAlgebra
```

Comment:
If we use good ol' `@deprecate_moved`, does that reintroduce a dependency on LinearAlgebra for the duration of the deprecation period?

Reply:
(not relevant at the moment, just something to consider for the Future)

Reply:
> does that reintroduce a dependency on LinearAlgebra for the duration of the deprecation period?

Maybe, but we can maybe just use a regular manually thrown `depwarn`. I'll add a commented out call to that in this PR.